### DISCUSSION TITLE
int8 vs int8_float16 compute type

### DISCUSSION THREAD

**Discussion by palladium123** [SYSTRAN/faster-whisper#173] [General]
When running on a GPU, are these identical?

**Comment by guillaumekln** [comment]
No they are different.

* With int8, non quantized layers are run in FP32.
* With int8_float16, non quantized layers are run in FP16.

See also the corresponding CTranslate2 documentation: https://opennmt.net/CTranslate2/quantization.html#supported-types

**Comment by christhegrand** [comment]
Is int8_float16 faster for inference?
